source/hosting/migrate-blob.txt

```diff
@@ -0,0 +1,199 @@
+.. _migrate-blob:
+
+Migrate to Your Own S3 Bucket
```

Comment:
One broad comment: I notice the GraphQL migration guides have a step called `Shut down MongoDB Atlas App Services`. Is there anything similar we should call out for the static hosting deprecation? Should we note either in these guides or on the main page what happens to the hosted content when the deprecation date arrives? i.e. "you'll no longer be able to serve your hosted content" or any other relevant details to note? (Does our hosting service store data in your linked Atlas data source? Should we advise people to delete their hosted data since they can't serve it anymore as of the deprecation date? I realize I have no idea how this works or what may be relevant to note for people as we are winding down this service.)

Reply:
Good call out – I agree this would be nice to have. We could add something similar to the GraphQL copy:

Shut Down Atlas Hosting
> Once you have verified that your application deploys successfully to [Vercel/Netlify], you can delete your hosted files from your Atlas App Services app. As a reminder, hosting domains on Atlas App Services will no longer run starting on March 12, 2025.